kube_controller_manager/metadata.csv

```diff
@@ -18,3 +18,9 @@ kube_controller_manager.nodes.count,gauge,,,,"Number of registered nodes, per zo
 kube_controller_manager.nodes.unhealthy,gauge,,,,"Number of unhealthy nodes, per zone",-1,kube_controller_manager,nodes.unhealthy
 kube_controller_manager.leader_election.transtions,count,,,,"Number of leadership transitions observed",0,kube_controller_manager,le.transitions
 kube_controller_manager.leader_election.lease_duration,gauge,,,,"Duration of the leadership lease",0,kube_controller_manager,le.duration
+kube_controller_manager.queue.process_duration.count,gauge,,,,"Process duration, by queue",0,kube_controller_manager,process.duration.count
```

Comment:
The short description is wrong, it shouldn't repeat the metric name. Don't fix the old ones, but it'd be nice to not introduce the new ones.